stellargraph/data/explorer.py

```diff
@@ -818,13 +818,16 @@ def run(
                 than or equal to the context window size.
             initial_edge_bias (str, optional): distribution to use when choosing a random
```

Comment:
Not that it makes any meaningful difference  but just for consistency's sake :-)

The descriptions of `num_cw (int)`, `cw_size (int)`, and `max_walk_length (int)`  start with  capital letters whereas, `initial_edge_bias (str, optional)` and `walk_bias (str, optional)` start with small letters. Also, full stops missing at a couple of places.

Reply:
Thanks, will update